packages/terra-table/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Changed background color to transparent
```

Comment:
Should this be a separate PR?

Reply:
it's here https://github.com/cerner/terra-core/pull/1758. looks like you branched off that branch instead of master

Reply:
Worked with @bjankord on reverting commit, should be correct now.